packages/emf/backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/services/ObjectService.java

```diff
@@ -246,14 +247,20 @@ public Optional<Object> getObject(IEditingContext editingContext, String objectI
     @Override
     public List<Object> getContents(IEditingContext editingContext, String objectId) {
         List<Object> contents = new ArrayList<>();
+        Optional<EObject> optEObject = this.getObject(editingContext, objectId)
+                .filter(EObject.class::isInstance)
+                .map(EObject.class::cast);
 
-        // @formatter:off
-        this.getObject(editingContext, objectId)
-        .filter(EObject.class::isInstance)
-                .map(EObject.class::cast)
-                .ifPresent(eObject -> contents.addAll(eObject.eContents()));
-        // @formatter:on
-
+        if (optEObject.isPresent()) {
```

Comment:
The copyright date of the file must be updated to 2023.

Minor: `optEObject` => `optionalEObject` (to be consistent with the rest of the code base).

Reply:
Minor too, but as long as you have to change the commit: "emf itemProvider" => "EMF ItemProvider" in the commit message.